Sources/MockoloFramework/Operations/OutputWriter.swift

```diff
@@ -37,8 +38,21 @@ func write(candidates: [(String, Int64)],
         break
     }
 
-    let importsSet = Set(importLines.map{$0.trimmingCharacters(in: .whitespaces)})
-    let importLineStr = importsSet.sorted().joined(separator: "\n")
+    var imports = importLines.compactMap { (importLine) -> String? in
+        return extractModuleImport(importLine: importLine.trimmingCharacters(in: .whitespaces))
```

Comment:
the extract function can live in stringextensions instead, then you can just call importLine.moduleName.